doc/jsk_pcl_ros/nodes/fuse_rgb_images.md

```diff
@@ -3,4 +3,4 @@
 ![](images/fuse_rgb_images.jpg)
 
 Fuse rgb images ignoring black region in each image.
-See [FuseDepthImages](fuse_depth_images.html) for other info.
+See [jsk_pcl_ros/FuseDepthImages](./fuse_depth_images.md) for other info.
```

Comment:
`./fuse_depth_images.md` -> `./fuse_depth_images.html`

Reply:
markdown document links will be compiled to html link.